src/components/list-toggle-button/list-toggle-button.tsx

```diff
@@ -0,0 +1,23 @@
+const {withText} = KalturaPlayer.ui.preacti18n;
+const {Icon} = KalturaPlayer.ui.components;
+import {Icon as IconPath} from 'types';
+
+import * as styles from './list-toggle-button.scss';
+
+const ListToggleButton = withText({
+  relatedVideosText: 'related.relatedVideos'
+})(({active, disabled, relatedVideosText}: {active: boolean; disabled: boolean; relatedVideosText: string}) => {
+  return (
+    <button
+      aria-label={relatedVideosText}
+      tabIndex={0}
+      disabled={disabled}
+      className={`${styles.listToggleButton} ${KalturaPlayer.ui.style.controlButton} ${KalturaPlayer.ui.style.upperBarIcon} ${
```

Comment:
why use use both classes controlButton and upperBarIcon They have two different purposes